engine/executor/openapi.go

```diff
@@ -43,13 +45,16 @@ func newJobAPIServer() *jobAPIServer {
 
 func (s *jobAPIServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if h, ok := s.match(r.URL.Path); ok {
+		log.Debug("serve http request", zap.String("path", r.URL.Path))
```

Comment:
@gozssky can the framework log OpenAPI request/response soon?

Reply:
I think it will be soon. This log can be kept here. If there is any better way, we can improve it later.